CHANGELOG

```diff
@@ -16,9 +16,8 @@
 * Sun Jul 15 2018 Chris Tessmer <chris.tessmer@onypoint.com> - 3.0.0-0
 - Removed support for TPM 2.0 (into its own module)
 
```

Comment:
Now there is no 2.0.0 entry so we're jumping from 1.2.1 to 3.0.0.  IMHO this is confusing, especially since 1.2.1 was the last released version